draft-ietf-mls-protocol.md

```diff
@@ -2013,18 +1995,6 @@ of the new group, provided the GroupId is long enough to prevent collisions.
 However, GroupId is currently 'hidden' in the GroupInfo, which is only available
 to the receiver _after_ the PSK would have to be injected. -->
 
-### Re-Initialization {#re-initialization}
```

Comment:
Why remove this, but not the subsection below on Sub-group branching? I think both section serve the purpose of giving an overview over the respective process and should be left in.

Reply:
This is all (correctly) discussed in the section on ReInit proposals. Sub-group branching isn't discussed anywhere else